docs/Text.md

```diff
@@ -0,0 +1,185 @@
+* Contextual forms - In some writing systems (eg. Arabic, Hebrew, Mongolian, etc.), characters may take a different shape, depending on whether they appear at the beginning, in the middle, or at the end of a word, or isolated. Fpdf will always use the same standard shape in those cases.
+* Vertical writing - Some writing systems are meant to be written vertically. Doing so is not directly supported. In cases where this just means to stack characters on top of each other (eg. Chinese, Japanese, etc.), client software can implement this by placing each character individuall at the correct location. In cases where the characters are connected with each other (eg. Mongolian), this may be more difficult, if possible at all.
+
+## Text Formatting
+For all text insertion methods, the relevant font related properties (eg. font/style and foreground/background color) must be set before invoking them. This includes using:
+* [`.set_font()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_font)
+* [`.set_text_color()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_text_color)
+* [`.set_draw_color()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_draw_color) - for cell borders
+* [`.set_fill_color()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_fill_color) - for the background
+
+In addition, some of the methods can optionally use [markdown](TextStyling.html#markdowntrue) or [HTML](HTML.html) markup in the supplied text in order to change the font style (bold/italic/underline) of parts of the output.
+
+## Change in current position
+`.cell()` and `.multi_cell()` let you specify where the current position (`.x`/`.y`) should go after the call. This is handled by the parameters `new_x` and `new_y`. Their values are one out of the Enums `.XPos` and `.YPos` respectively, which offer the following options:
+
+#### .XPos
+* XPos.LEFT    - left end of the cell
+* XPos.RIGHT   - right end of the cell
+* XPos.START   - start of actual text
+* XPos.END     - end of actual text
+* XPos.WCONT   - for write() to continue next (slightly left of END)
+* XPos.CENTER  - center of actual text
+* XPos.LMARGIN - left page margin (start of printable area)
+* XPos.RMARGIN - right page margin (end of printable area)
+
+#### .YPos
+* YPos.TOP     - top of the first line
+* YPos.LAST    - top of the last line (same as TOP for single-line text)
+* YPos.NEXT    - top of next line (bottom of current text)
+* YPos.TMARGIN - top page margin (start of printable area)
+* YPos.BMARGIN - bottom page margin (end of printable area)
+
+
+## .text()
+Prints a single-line character string. In contrast to the other text methods,
+the position is given explicitly, and not taken from `.x`/`.y`. The origin is
+on the left of the first character, on the baseline. This method allows placing
+a string with typographical precision on the page, but it is usually easier to
+use the `.cell()`, `.multi_cell()` or `.write()` methods.
+
+#### text(x, y, txt='')
```

Comment:
I'd prefer to remove this paragraph and simply points to https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.text

Reply:
Ah yes, with all of them, I can see your point now.

(Currently travelling with a wonky laptop keyboard, so not looking forward to any non-trivial changes 🙄 )